apistar/test.py

```diff
@@ -77,7 +77,9 @@ def send(self, request, *args, **kwargs):
         """
         raw_kwargs = {}
 
-        def start_response(wsgi_status, wsgi_headers):
+        def start_response(wsgi_status, wsgi_headers, exc_info=None):
+            if exc_info:
+                raise exc_info[0].with_traceback(exc_info[1], exc_info[2])
```

Comment:
Should this instead be `raise exc_info[0](exc_info[1]).with_traceback(exc_info[2])`?

Eg. see https://docs.python.org/3/library/exceptions.html#BaseException.with_traceback and https://docs.python.org/3.6/library/sys.html#sys.exc_info

Reply:
Also we *might* want to consider a test case.